firmware/boron-ins-fsm/src/flashAddresses.h

```diff
@@ -34,6 +34,10 @@
 #define ADDR_STATE3_MAX_LONG_STILLNESS_TIME                 23  // uint32_t = 4 bytes
 #define ADDR_INITIALIZE_STATE3_MAX_LONG_STILLNESS_TIME_FLAG 27  // uint16_2 = 2 bytes
 
-// next available address is 27 + 2 = 29
+// new state machine constant for state0 window
+#define ADDR_INITIALIZE_STATE0_WINDOW_FLAG                  29  // uint16_2 = 2 bytes
```

Comment:
Looks like your changes are failing on Travis due to linting errors: https://app.travis-ci.com/github/bravetechnologycoop/BraveSensor/jobs/610053932

Take a look in the README on how to run the `clang` formatting scripts. There's one that will give you the list of problems and another that will actual fix the problems for you. You should be able to run this on your computer.